python/python/tests/test_async_client.py

```diff
@@ -795,6 +795,87 @@ async def test_hkeys(self, redis_client: TRedisClient):
         with pytest.raises(RequestError):
             await redis_client.hkeys(key2)
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_hrandfield(self, redis_client: TRedisClient):
+        key = get_random_string(10)
+        key2 = get_random_string(5)
+        field = get_random_string(5)
+        field2 = get_random_string(5)
+        field_value_map = {field: "value", field2: "value2"}
+
+        assert await redis_client.hset(key, field_value_map) == 2
+        rand_field = await redis_client.hrandfield(key)
+        assert rand_field == field or rand_field == field2
```

Comment:
nit
```suggestion
        assert rand_field in [field, field2]
```
or even merge with previous line